dev-python/pygobject/pygobject-3.42.2.ebuild

```diff
@@ -29,7 +29,7 @@ RDEPEND="
 DEPEND="
 	${RDEPEND}
 	test? (
-		dev-libs/atk[introspection]
+		>=app-accessibility/at-spi2-core-2.46.0[introspection]
```

Comment:
Any reference/explanation for this change?

Reply:
package.deprecated, and I should have indicated this in the commit message.